Foundation/NSURLResponse.swift

```diff
@@ -396,8 +396,8 @@ private extension String {
                 type = string
             } else {
                 if let r = string.range(of: "=") {
-                    let name = string[string.startIndex..<r.lowerBound].trimmingCharacters(in: ws)
-                    let value = string[r.upperBound..<string.endIndex].trimmingCharacters(in: ws)
+                    let name = String(string[string.startIndex..<r.lowerBound]).trimmingCharacters(in: ws)
```

Comment:
This is fairly unfortunate that we have to force a copy here, In the grand scheme of things it probably won't matter too much since the range is probably always small